src/app.constants.js

```diff
@@ -26,6 +26,7 @@ export const MOVIE_TYPE = {
 
 export const PARAMS = {
   url: 'https://api.themoviedb.org/3/movie/',
+  genres_url: 'https://api.themoviedb.org/3/genre/movie/list',
```

Comment:
please don't duplicate this part - https://api.themoviedb.org/3/
create separate constant BASE_URL for it